### DISCUSSION TITLE
Page load error during the exam

### DISCUSSION THREAD

**Discussion by bonbon-2** [SafeExamBrowser/seb-win-refactoring#474] [Q&A]
Hi,

![IMG-20220926-WA0000](https://user-images.githubusercontent.com/94406273/192552021-f91ca5cd-8082-4057-9a45-f34e128f0ab5.jpg)

Quite often we get an page load error during the exam. Sometimes pressing OK several times, makes the error message disappear.
Other times we have to pauze the exam, because the computer does nothing anymore. 
Force the program to end, and start the program again.

We have the problem on multiple locations. I can't figure out if it is the (wired) internet connection, the exam program or SEB. 
I just know it's very annoying!

We use Windows 11 pro machines with Safe Exam Browser, Version 3.3.2 (x64), Build 3.3.2.413

Does anyone recognize this error message or know a solution?

**Comment by dbuechel** [comment]
This does look like a network issue on your side. If it were a general issue with SEB, then there would be a lot of similar reports for version 3.3.2.

**Comment by bonbon-2** [comment]
OK, thanks. 
The next exam I will use our wireless network connection, see if we get rid of the page load error
